main

```diff
@@ -0,0 +1,85 @@
+/usr/bin/env ruby
```

Comment:
- [ ] First i was unable to run your application because you saved this file as `main` instead of `main.rb` so please kindly correct this
- [ ] Also when i corrected the naming of the file i was still unabale to run your project because of this highlighted piece of code on line 1. However once i removed i was able to see the menu which starts the app. please kindly make these corrections so that the next reviewer can ultimately be able to test your project for this milestone :+1: 
- [ ] You did a good job with your code base by the way :+1: :sparkles: